infra/Pos/Slotting/Impl/Ntp.hs

```diff
@@ -184,19 +185,19 @@ ntpGetCurrentSlotInaccurate var = do
             ntpCurrentTime var >>= approxSlotUsingOutdated penult
 
 ntpGetCurrentSlotImpl
-    :: NtpMode m
-    => NtpSlottingVar -> m SlotStatus
+    :: (NtpMode m, MonadThrow m)
+    => NtpSlottingVar 
+    -> m SlotStatus
 ntpGetCurrentSlotImpl var = do
     NtpSlottingState {..} <- atomically $ STM.readTVar var
     t <- Timestamp . (+ _nssLastMargin) <$> currentTime
     case canWeTrustLocalTime _nssLastLocalTime t of
       Nothing -> do
-          penult <- sdPenultEpoch <$> getSlottingData
+          lastIndex <- getEpochLastIndex
           res <- fmap (max _nssLastSlot) <$> slotFromTimestamp t
-          let setLastSlot s =
-                  atomically $ STM.modifyTVar' var (nssLastSlot %~ max s)
+          let setLastSlot s = atomically $ STM.modifyTVar' var (nssLastSlot %~ max s)
           whenJust res setLastSlot
-          pure $ maybe (OutdatedSlottingData penult) CurrentSlot res
+          pure $ maybe (OutdatedSlottingData lastIndex) CurrentSlot res
```

Comment:
I see you changed the semantics from attaching penult index to last index, but didn't change a comment to `OutdatedSlottingData` and the place where it's printed.